spec/PasswordPolicy.spec.js

```diff
@@ -29,7 +29,11 @@ describe("Password Policy: ", () => {
       user.set('email', 'user@parse.com');
       return user.signUp();
     }).then(user => {
-      Parse.User.requestPasswordReset("user@parse.com");
+      Parse.User.requestPasswordReset('user@parse.com').catch((err) => {
```

Comment:
@bhaskaryasa 
I think we should use the DRY principle here as in if there is a another place where this kind of `requestPasswordReset` test is already tested then you don't need to test again in your code.

This will make the whole testing process much faster.

In this case `requestPasswordReset` is tested in test cases outside of your code already. Those basic test cases will fail if there is a problem with your code.  That is the reason I was saying we can catch it in the one catch block you have.

thoughts?